configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg

```diff
@@ -0,0 +1,27 @@
+#
+# Configure a cache server for the CMS Data Federation.
+#
+# **********************************************************************
+# * WARNING: DO NOT EDIT THIS FILE.  IT WILL BE OVERWRITTEN ON UPGRADE *
+# **********************************************************************
+#
+# Instead of editing this file,
+# - Edit `/etc/xrootd/config.d/10-common-site-local.cfg` to provide a few
+#   MANDATORY variable values.
+# - Override OSG-provided defaults by dropping a site-specific configuration
+#   file in `/etc/xrootd/config.d` and prefixing it with "90-" or higher.
+#
+# This file is part of the StashCache Daemon
+# https://opensciencegrid.org/docs/data/stashcache/overview/
+#
+
+all.role server
+all.manager xrootd.t2.ucsd.edu:2041
```

Comment:
This should be removed and also as an env variable.

Reply:
What is `all.manager` exactly? What guidance can we provide to admins for what this should point to?

Reply:
Well is the redirector i you have a logical cached formed of several machines

Reply:
Does CMS want all caches to be behind a redirector or is it optional?

Reply:
Well the thing is that if you want more than one node for disks reasons (there are only so many disks basys you can fit on a box) each site will have to have a cache redirector.

Reply:
That makes sense but I'm interested in what the 'default' recommendation is, i.e. if a CMS site were to set-up a completely new XCache, what would that look like? Something like this?

- A redirector (e.g. a host running the cmsd and xrootd services using the config we're shipping here)
- A couple of hosts running just the xrootd service

Reply:
Well idk but lets say all NANOAOD is like 50-100TB with 10TB disks this means 10 bays most 4U chasis have more than enough space for this. So I would say for CMS standalone no redirector at all and we add extra docuentation for redirector.

Reply:
So in the standalone case they wouldn't even need to run `cmsd` on their XCache host, right?

Reply:
That is ocrrect but I would still have cmsd to make the horizontable scalling less disruptive in the future

Reply:
I've removed it from the default config and added a new config file for this: https://github.com/opensciencegrid/xcache/blob/c704e862e7b240948c1a6ed259270fc51c32a979/configs/cms-xcache/config.d/90-cms-xcache-local-redirector.cfg